projects/amun/index.js

```diff
@@ -1,41 +1,135 @@
-const sdk = require('@defillama/sdk');
-const abi = require('./abi.json')
+const sdk = require("@defillama/sdk");
+const { default: computeTVL } = require("@defillama/sdk/build/computeTVL");
+const { util } = require("@defillama/sdk");
+const abi = require("./abi.json");
+const axios = require("axios");
+const solana = require("../helper/solana");
+const utils = require("../helper/utils");
+const BigNumber = require("bignumber.js");
 
-const DFI_ADDRESS = '0xA9536B9c75A9E0faE3B56a96AC8EdF76AbC91978';
-const DMX_ADDRESS = '0x1660F10B4D610cF482194356eCe8eFD65B15bA83';
+const PECO_ADDRESS = "0xA9536B9c75A9E0faE3B56a96AC8EdF76AbC91978";
+const DFI_ADDRESS = "0xA9536B9c75A9E0faE3B56a96AC8EdF76AbC91978";
+const SOLI_ADDRESS = "8JnNWJ46yfdq8sKgT1Lk4G7VWkAA8Rhh7LhqgJ6WY41G";
+
+const knownTokenPrices = {};
+
+async function getSolanaTokenRate(address) {
+  return await axios.get(
+    "https://lima-api.amun.com/tokenRate/APygy51vntbQQsYmgkF1CeMsAn5Tv1rsVUB6nee3wJpY?chain=solana&address=" +
+      address
```

Comment:
`https://lima-api.amun.com/tokenRate/${address}?chain=solana`, where address is a program Id (?) for this token - `APygy51vntbQQsYmgkF1CeMsAn5Tv1rsVUB6nee3wJpY` for SOLI

Reply:
yes - https://amun.atlassian.net/wiki/spaces/EN/pages/347898072/SOLI+version+1.0.0